gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1077,7 +1077,8 @@ gst_tensor_filter_configure_tensor (GstTensorFilter * self,
         /** if set-property called and already has info, verify it! */
         if (prop->output_meta.num_tensors > 0) {
           if (!gst_tensors_info_is_equal (&prop->output_meta, &out_info)) {
-            gchar *str = _compare_tensors (&in_config.info, &prop->input_meta);
+            gchar *str =
+                _compare_tensors (&out_config.info, &prop->output_meta);
```

Comment:
In this line, out_config is not updated yet. Please change to &out_info.
_compare_tensors (&out_info, &prop->output_meta);

Reply:
it's applied :)